introduction.md

```diff
@@ -0,0 +1 @@
+Hola I am Mind Cookin, a humble web and game developer who wants to learn GitHub, and have fun.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.